src/data/tokenStore/index.ts

```diff
@@ -25,5 +28,12 @@ const getTokenFilePath = (tokenFile: string) => {
   return tokenFilePath;
 };
 
-store = createFileTokenStore(getTokenFilePath(Constants.AppCenterTokenFileName));
+if (os.platform() === "win32") {
+  store = createWinTokenStore();
+} else if (os.platform() === "darwin") {
+  store = createOsxTokenStore();
+} else {
+  store = createFileTokenStore(getTokenFilePath(Constants.AppCenterTokenFileName));
+}
 export const tokenStore = store;
+export const fileTokenStore = createFileTokenStore(getTokenFilePath(Constants.AppCenterTokenFileName));
```

Comment:
why export again `fileTokenStore`?  On line 38 we export `tokenStore`, do we need additional export?

Reply:
Yep for compatibility